AUTHORS.rst

```diff
@@ -7,33 +20,20 @@ contributors to this package were
     Pavol Juhas
     Dmitriy Bryndin
 
-The DiffPy team is located in the Billinge-group at the Applied Physics
```

Comment:
Let's keep this in but change the wording a bit, tipping our hat to the history.